specs/CoreWebView2.CreateFromCOMObject.md

```diff
@@ -47,8 +53,52 @@ public class MyWebView2Control
 }
 ```
 
+## WinRT to COM
+
+```c++
+winrt::com_ptr<ICoreWebView2> GetComICoreWebView2FromCoreWebView2(
+    winrt::Microsoft::Web::WebView2::Core::CoreWebView2 coreWebView2WinRT)
+{
+    // Get the COM interop interface from the WinRT CoreWebView2.
+    auto interop = coreWebView2WinRT.as<ICoreWebView2Interop2>();
+
+    // Get the COM ICoreWebView2 object from the COM interop interface.
+    winrt::com_ptr<ICoreWebView2> coreWebView2Com;
+    winrt::check_hresult(interop->GetComICoreWebView2(coreWebView2Com.put()));
+	
+	return coreWebView2Com;
+}
+```
+
+## COM to WinRT
+
+```c++
+winrt::Microsoft::Web::WebView2::Core::CoreWebView2 GetCoreWebView2FromComICoreWebView2(
```

Comment:
Should we name it CreateCoreWebView2FromComICoreWebView2 instead of GetCoreWebView2FromComICoreWebView2?